format.md

```diff
@@ -141,6 +141,11 @@ Polygon.
 Truly mixed geometry types can be supported as a union of the other geometry
 types, and it is planned to add a description of this later.
 
+### Field and child names
+
+All geometries should name the fields and children as suggested for each geometry type,
```

Comment:
This first part reads a bit strange (it's not the geometries that "name" the fields? It's the implementation that does that, or the geometries _have_ field names as suggested for each geometry type?)

Reply:
Good point, I attempted to improve the wording in f70ef197ff1f046be2b3454abf73de937e1c7c48.